### PR TITLE
Topic recognition

### PR DESCRIPTION
**Pull request by TungChinYang**
This is the pull request from student in COMP3710.

I worked on Brain MRI Segmentation using improved Unet.

Student nember:4565256
Student name: Tung-Chin Yang

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Commit messages should be a lot more specific
- Code valid for Easy Difficulty
- Only 4 DSC evaluated? we need the entire test set. 
- Results appear to be good

Other Feedback
- Give your main script a more meaningful name so people know which one to run.

**Comment by rexsw** [comment]
Code looks good for easy difficulty

For next time;

Tensorflow has functions for image resizing and more so there’s no need to also use PIL in your project. 

Paths such as "C:/Users/s4565256/Downloads/keras_png_slices_data/keras_png_slices_test/*.png" won’t work for the user and should either be set dynamical or it needs to be made clear to the user that they need to change these paths

Code needs meaningful comments

Code in files such as fitmodel.py should be in a main function in a python file.

Make sure you remove unused imports

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme OK
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
**Feedback required. remove unnecessary files.**
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info. 
Notebooks have markdown sections for you to explain what's going on, so you should make use of it.

**Comment by TungChinYang** [comment]
Hi Siyu,

I can’t find my pull request anymore. Is there any problem?

Sincerely 
Tung-Chin

**Comment by SiyuLiu0329** [comment]
Don’t worry, your pull request has been merged into master, it is part of master now, what’s why you don’t see it.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
